docs/developer-docs-home.md

```diff
@@ -2,31 +2,61 @@
 
 ## Table of contents
 
-1. [Documentation index](#documentation-index)
+1. [What is a field plug-in](#what-is-a-field-plug-in)
+1. [What can a field plug-in do](#what-can-a-field-plug-in-do)
+1. [How does a field plug-in interact with a form](#how-does-a-field-plug-in-interact-with-a-form)
 1. [How to start](#how-to-start)
+1. [Documentation index](#documentation-index)
 1. [Sharing your field plug-ins](#sharing-your-field-plug-ins)
 1. [Useful field plug-ins for developers](#useful-field-plug-ins-for-developers)
 
-## Documentation index
+## What is a field plug-in
 
-* [Field plug-in definition](https://github.com/surveycto/field-plug-in-resources/blob/master/docs/plug-in-definition.md)  
-    This document describes the structure and format of field plug-ins. It is recommended that you read this first.  
-* [API reference](https://github.com/surveycto/field-plug-in-resources/blob/master/docs/api-reference.md)  
-    This document contains a comprehensive list of available APIs that your field plug-in can use to interact with the form.  
-* [Using field plug-ins (user documentation)](https://docs.surveycto.com/02-designing-forms/03-advanced-topics/06.using-field-plug-ins.html)  
-    This article is written for form designers and will explain how to install and use field plug-ins in SurveyCTO.
+A field plug-in can be thought of as a web page. When filling out a SurveyCTO form, this web page will load instead of the standard field view. A field plug-in effectively 'takes over' the field on which it is used. In this way, field plug-ins enable you to have complete control over the appearance and functionality of questions in your data collection instrument.
+
+If you've ever created your own web page using HTML, CSS, and Javascript, then you already understand the underlying technology. A field plug-in uses those three basic web technologies to create a web view whenever the field is loaded in a form. You can use all the features of standard Javascript to control what happens in the field plug-in itself, then use our form APIs to control how your field plug-in interacts with the rest of the form.
+
+## What can a field plug-in do
+
+The short answer is that a field plug-in can do anything that a normal web page can do. When filling out a SurveyCTO form and you arrive at a field that uses a field plug-in, a web page is created using your field plug-in's code. On Android, a field plug-in is loaded in its own fully-functional Android webview. On iOS, it is loaded in a WKWebView. In web forms, it is loaded in an iframe. No matter which platform is used, a field plug-in is free to use HTML, CSS, and Javascript in the same way any normal web page can.
```

Comment:
```suggestion
The short answer is that a field plug-in can do anything that a normal web page can do. When filling out a SurveyCTO form and you arrive at a field that uses a field plug-in, a web page is created using your field plug-in's code. On Android, a field plug-in is loaded in its own fully-functional Android WebView. On iOS, it is loaded in a WKWebView. In web forms, it is loaded in an iframe. No matter which platform is used, a field plug-in is free to use HTML, CSS, and Javascript in the same way any normal web page can.
```

Seems worth capitalizing WebView to match the proper class-name capitalization of WkWebView.